v1.0/anatomicalEntity.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/AnatomicalEntity",
+  "description": "Structured information describing an anatomical entity (term).",
+  "required": [
+    "nameOfLocation"
+  ],
+  "properties": {
+    "nameOfLocation": {
+      "type": "string",
+      "_instruction": "Enter a descriptive name for this anatomical entity based on anatomical location or characteristics."
+    },    
+    "otherAnatomicalRelations": {
+      "type": "array",
+      "_instruction": "Add one or several anatomical entity relation(s) to this anatomical entity describing its relation to other anatomical entities.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/sands/AnatomicalEntityRelation"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "parentStructure": {
+      "_instruction": "Add another anatomical entity to which this anatomical entity is part of.",
```

Comment:
I don't think I get what you mean by that. If they are non-hierarchical, it wouldn't be a parent structure anymore, would it?

Reply:
It would be useful to have an example of a non-hierarchical terminology that would use the parent-function.
I can't think of an example myself.

Reply:
There are some brain regions that can be associated to more than one parent structure because they are exactly at the border of for example two lobes. In some terminologies they do not decide for one but name this structure twice (for each lobe). I would need to check in more detail for a concrete example, I don't remember anymore when I encountered that issue.

Reply:
Discussed with @lzehl: We will try without non-hierarchical first (count = 1).